quinn/src/endpoint.rs

```diff
@@ -304,6 +321,16 @@ impl Endpoint {
     }
 }
 
+// Struct for holding the stats returned from Endpoint::stats
+//
+#[derive(Debug, Default, Copy, Clone)]
+pub struct EndpointStats {
+    pub open_connections: u64,
```

Comment:
Would it be more useful/consistent to lean into the monotonic counter paradigm and count the lifetime number of successfully established and lost connections, rather than reporting the instantaneous live count? I have limited experience with large-scale infrastructure monitoring so maybe I'm getting carried away.

Reply:
I think we would find it more useful to just sample the number of open connection at that moment.

Reply:
Would that be adequately addressed by using the existing [`Endpoint::open_connections`](https://docs.rs/quinn-proto/latest/quinn_proto/struct.Endpoint.html#method.open_connections) rather than committing to it as part of `EndpointStats`?

Reply:
Don't really have a strong opinion on this. I just thought it was something we might as well include in the stats interface - maybe eventually superseding [Endpoint::open_connections](https://docs.rs/quinn-proto/latest/quinn_proto/struct.Endpoint.html#method.open_connections) though you will know better than me whether this is appropriate.